internal/broker/broker_test.go

```diff
@@ -729,6 +732,7 @@ func TestConcurrentIsAuthenticated(t *testing.T) {
 
 			b := newBrokerForTests(t, &brokerForTestConfig{
 				Config:          broker.Config{DataDir: dataDir},
+				allUsersAllowed: true,
```

Comment:
```suggestion
				anylUserAllowed: true,
```
maybe?

Reply:
I would rather keep a consistent name across the code, unless you want me to change the name of the field in `userConfig` as well. BUT then we would be mapping the `ALL` keyword to `any`, which again feels weird.

Reply:
Ok, fair enough